ansible/host_vars/linux-01.ih-eu-mda1.nimbus.sepolia.yml

```diff
@@ -21,6 +21,15 @@ redirect_ports:
 era_files_domain: 'sepolia.era.nimbus.team'
 era_files_path: '/data/era'
 
+# Era1 files hosting
+era1_files_domain: 'sepolia.era1.nimbus.team'
+era1_files_path: '/data/era1'
+
+# Open Nginx Ports
+open_ports_default_comment: 'Nginx'
+open_ports_default_chain: 'SERVICES'
+open_ports_list: [ { port: 80 }, { port: 443 } ]
```

Comment:
You don't need this, you're overriding the values set here this way, which is bad:
https://github.com/status-im/infra-nimbus/blob/0c26a3a8f1fc20ae25378fa4d2153e172173078b/ansible/group_vars/nimbus.sepolia.yml#L111-L122